drivers/ninaw10/machine_pin_nina.c

```diff
@@ -55,6 +57,16 @@ static uint8_t pin_map[MICROPY_HW_PIN_EXT_COUNT] = {
     35, // A7
 };
 
+static uint8_t adc_map[MICROPY_HW_PIN_EXT_COUNT] = {
```

Comment:
Would be good to add a comment explaining what numbers this maps from and what it maps to.